src/app/core/component/markdown-content-container/markdown-pattern-sectioncontent/markdown-pattern-section-content.component.ts

```diff
@@ -69,11 +69,15 @@ export class MarkdownPatternSectionContentComponent extends DataRenderingCompone
     this.changeText(this.renderedData);
   }
 
-  changeText(value: string): void {
+  changeText(value: string | undefined): void {
     this.markdownDiv.nativeElement.innerHTML = '';
-    this.renderedData = value;
+    if(value === undefined) {
+      this.renderedData = '';
+    } else {
+      this.renderedData = value;
+    }
```

Comment:
```suggestion
    this.renderedData = value ?? "";
```
Use `??` to provide default values that will be used if the left side is nullish (null or undefined)